### DISCUSSION TITLE
推迟了？

### DISCUSSION THREAD

**Discussion by renwofei423** [web-infra-dev/modern.js#75] [General]
从10月14推迟到27 日了啊   等得好着急啊

**Comment by dexteryy** [comment]
嗯，为了配合稀土开发者大会，改成 10.27 发布，顺便也有更多时间完善文档和测试

官网 ：https://modernjs.dev/

稀土开发者大会 ：https://conf.juejin.cn/xdc2021?utm_source=mdjs
